### PR TITLE
Update native-file-drag-drop.md

### PR DESCRIPTION
**Pull request by dengjie13520**
electronic 12 + needs to set contextisolation to false

### CODE REVIEW

**Review by MarshallOfSound — CHANGES_REQUESTED** [review]
The example should be updated to work with `contextIsolation: true` rather than suggesting an insecure option be used

**Comment by miniak** [comment]
@dengjie13520 also FYI it is Electron not "electronic"

**Comment by codebytere** [comment]
@dengjie13520 would you like to move this PR forward?

File: docs/tutorial/native-file-drag-drop.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ document.getElementById('drag').ondragstart = (event) => {
   ipcRenderer.send('ondragstart', '/absolute/path/to/the/item')
 }
 ```
+Note: electronic 12 + needs to set contextisolation to false
```
Comment (GrahamSH-LLK) [inline docs/tutorial/native-file-drag-drop.md:36]: You mean Electron, afaik. Also, there should be a period at the end.

**Comment by codebytere** [comment]
Closing as no response has been given. Happy to consider re-opening though.

**Pull request closed without merging**